oauthenticator/generic.py

```diff
@@ -84,10 +84,18 @@ class GenericOAuthenticator(OAuthenticator):
         help="Disable basic authentication for access token request",
     )
 
+    ca_certs = Unicode(
+        os.environ.get('OAUTH2_CA_CERTS', os.environ.get('SSL_CERT_FILE')),
```

Comment:
I'm not sure if defaulting to `SSL_CERT_FILE` is a good idea if `OAUTH2_CA_CERTS`

 I'm thinking of a scenario where `SSL_CERT_FILE` is set to a path and `OAUTH2_CA_CERTS` should be a different one, like the default system cert location. Can something like this be possible?

Reply:
@GeorgianaElena I did scan over similar tornado issues/posts and they claim they do provide the option to override the CA certs and put the onus on the client code to handle setting that properly, hence this PR.

Maybe there is a better place to do this in the oauth2, but I couldn't find it and came up with this solution to allow me to override ca-certs to get this working in my scenario.

I have implemented it to first use a new `OAUTH2_CA_CERTS` env (higher precedence) with fallback to `SSL_CERT_FILE` env (b/c that env is what is used by the python ssl module). I don't think `SSL_CERT_FILE`  can be a directory, if you want I can remove this fallback, in the end of both of those environment variables are not set, then it is the save behavior as originally.